src/node_crypto.cc

```diff
@@ -1816,17 +1816,6 @@ void SSLWrap<Base>::LoadSession(const FunctionCallbackInfo<Value>& args) {
     if (w->next_sess_ != nullptr)
       SSL_SESSION_free(w->next_sess_);
     w->next_sess_ = sess;
```

Comment:
I just left the error case silently ignoring it. Happy to change it if you'd prefer something else.